faker/providers/ssn/fr_CH/__init__.py

```diff
@@ -31,3 +31,20 @@ def _checksum(digits):
                          + digits[7:11] + '.' \
                          + digits[11:]
         return ssn
+
+    def vat_id(self):
+        """
+        :return: Swiss UID number
+        """
+        def _checksum(digits):
+            code = ['8', '6', '4', '2', '3', '5', '9', '7']
+            digits = [*digits]
```

Comment:
what's the purpose of making a copy of the argument and assigning it the same name?

Reply:
I'm new to python (about 2.5 months since I first touched a piece of Python code) 
I thought this was for converting the string into an array, but now that you say it, I guess python treats strings as arrays of chars anyways and this isn't actually needed.